proto/api/bria.proto

```diff
@@ -184,6 +184,19 @@ message CreateBatchGroupResponse {
   string id = 1;
 }
 
+message BatchGroup {
+  string id = 1;
+  string account_id = 2;
```

Comment:
Don't include `account_id`. In the regular api the account scope is implicit. We don't expose it. Only the admin api cares about account_id.